src/dev/src/app/wizard/wizard-alt-next.demo.html

```diff
@@ -26,7 +26,7 @@
 
   <clr-wizard-page (clrWizardPageNext)="pageCustomNext()" [clrWizardPagePreventDefaultNext]="true">
     <ng-template clrPageTitle>Page level alt-next</ng-template>
-    <p *ngIf="showCancelConfirm">Complete this fibonacci sequence</p>
+    <p>Complete this fibonacci sequence</p>
```

Comment:
Why are we getting rid of the `ngIf` if we are keeping the toggling of the `showCancelConfirm` property in `wizard-alt-cancel.demo.ts`?

Reply:
I removed `showCancelConfirm` in `wizard-alt-next.demo.html` since it was not referenced anywhere else in the template or `wizard-alt-next.demo.ts` so it was always hidden. The `showCancelConfirm` property in `wizard-alt-cancel.demo.ts` was being used in multiple places to demo the cancel functionality.